lib/galaxy/jobs/runners/univa.py

```diff
@@ -1,6 +1,7 @@
 """
-Kind of fail save job control via the DRMAA API.
-Tested for UNIVA grid engine
+Kind of fail save job control via the DRMAA API / qstat and qacct.
```

Comment:
fail-safe ?

Reply:
Don't know how this got there :) .. consider it gone.